docs/glossary/content-type.md

```diff
@@ -5,5 +5,10 @@ eleventyNavigation:
   parent: Glossary
 title: Content type
 theme: Information layer
-details: Test
----
\ No newline at end of file
+---
+A content type is a reusable container for organising content that has a common structure and purpose. For example, if you had a group of content items that are all types of speech, you'd create a 'speech' content type so they share a consistent structure and classification.
```

Comment:
Is `reusable container` the right term here? I thought it was more like `Content types are a way of organising content that has a common...`

Reply:
Updated by @sallycreasey